graphtools/base.py

```diff
@@ -656,13 +766,19 @@ def to_pickle(self, path):
         path : str
             File path where the pickled object will be stored.
         """
-        if int(sys.version.split(".")[1]) < 7 and isinstance(self, pygsp.graphs.Graph):
+        if int(
```

Comment:
These line breaks are ugly as sin. Can you put the line break after the `and`? You'll need to put the whole thing in parentheses to do that but I'd rather than than have this if statement span four lines.

Reply:
Nope

Reply:
Ok I fixed this in the new commit with a refactor using a shallow copy of the object to be pickled.